lib/state/index.js

```diff
@@ -11,11 +11,13 @@ import persistState from 'redux-localstorage';
 import appState from '../flux/app-state';
 
 import auth from './auth/reducer';
+import revisions from './revisions/reducer';
 import settings from './settings/reducer';
 
 export const reducers = combineReducers( {
 	appState: appState.reducer.bind( appState ),
 	auth,
+	revisions,
```

Comment:
It sounds like revisions are properties of notes, but in this case they are only part of the revision selector view.
While having them always available seems awesome, wouldn't it be quite heavy performance-wise?